contrib/backup/functions

```diff
@@ -51,9 +53,7 @@ function backup_db () {
     ln -sfn ${BACKUP_DIR}/${TIMESTAMP}_zammad_db.mysql.gz ${BACKUP_DIR}/latest_zammad_db.mysql.gz
   elif [ "${DB_ADAPTER}" == "postgresql" ]; then
     echo "creating postgresql backup..."
-    su -c "pg_dump --no-privileges --no-owner -c ${DB_NAME} | \
-      gzip > /tmp/${TIMESTAMP}_zammad_db.psql.gz" postgres
-    mv /tmp/${TIMESTAMP}_zammad_db.psql.gz ${BACKUP_DIR}
+    su -c "pg_dump --no-privileges --no-owner ${DB_HOST:+-h $DB_HOST} ${DB_PORT:+-p $DB_PORT} ${DB_USER:+-U $DB_USER} -d ${DB_NAME}" zammad | gzip > ${BACKUP_DIR}/${TIMESTAMP}_zammad_db.psql.gz
```

Comment:
I guess the original command works if Zammad and PostgreSQL are running on the same host **and** you're running the backup script as `root`. I think it should really be the `zammad` user running `pg_dump`, like the restore command below.

The rest is adding values for DB host, port, and user, if present, to allow for a non-local DB.